src/constants/DISPUTE_CATEGORY.ts

```diff
@@ -0,0 +1,10 @@
+export const DISPUTE_CATEGORY = [
+  { value: 'SCOPE_AND_CONTRACT_DISPUTES_ISSUES', label: 'Scope and Contract Disputes Issues' },
+  { value: 'INCOMPLETE_OR_UNSATISFACTORY_WORK', label: 'Incomplete or Unsatisfactory Work' },
+  { value: 'COMMUNICATION_PROBLEMS', label: 'Communication Problems' },
```

Comment:
1. Isn't it better to get categories from API? 
2. I think not all categories are allowed for all users. for example, talent shouldn't be able to select `Incomplete or Unsatisfactory Work` as a category while initiating a dispute.

Reply:
1. it's static and it is not huge data, no need for API (I always prefer to reduce API calls)
2. no one told me about it and Minh designed one category list for both sides